### ISSUE TITLE
is there a way to achieve the same as tail -f /var/log/syslog? with some search term?

### ISSUE THREAD

**Issue by 50hz** [ekanite/ekanite#88]
is there a way to achieve the same as tail -f /var/log/syslog? with some specific bleve search term? thanks

**Comment by otoolep** [comment]
No, streaming the results back is not currently supported. You'd need to write you own code that would periodically send the query to the system, and display new results.

**Comment by otoolep** [comment]
Not a bad idea for a feature, though it would be major work. Something to think about.

**Comment by 50hz** [comment]
thanks a lot! it is definitely a good idea for new feature :)

**Comment by tsenart** [comment]
This would be great :)

**Comment by mhfrantz** [comment]
Duplicate of #2 perhaps?
